bridges/otelzap/encoder_test.go

```diff
@@ -65,6 +65,13 @@ func TestObjectEncoder(t *testing.T) {
 			},
 			expected: []interface{}{wantTurducken, wantTurducken},
 		},
+		{
+			desc: "AddReflected",
+			f: func(e zapcore.ObjectEncoder) {
+				assert.NoError(t, e.AddReflected("k", map[string]interface{}{"foo": 5}), "Expected AddReflected to succeed.")
+			},
+			expected: "{\"foo\":5}\n",
```

Comment:
It would be great if this would pass:

```suggestion
			expected: map[string]interface{}{"foo": 5},
```

(similar comment bellow)